authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerPathBaseMetadataObject.java

```diff
@@ -42,17 +42,6 @@ public enum Type implements AuthorizationMetadataObject.Type {
     public MetadataObject.Type metadataObjectType() {
       return metadataType;
     }
-
-    public static RangerHadoopSQLMetadataObject.Type fromMetadataType(
-        MetadataObject.Type metadataType) {
-      for (RangerHadoopSQLMetadataObject.Type type : RangerHadoopSQLMetadataObject.Type.values()) {
-        if (type.metadataObjectType() == metadataType) {
-          return type;
-        }
-      }
-      throw new IllegalArgumentException(
-          "No matching RangerMetadataObject.Type for " + metadataType);
-    }
   }
 
   private final String path;
```

Comment:
Can we rename this class from `RangerPathBaseMetadataObject` to `RangerPathBasedMetadataObject` or `RangerPathMetadataObject`?

Reply:
OK, I rename `RangerPathBaseMetadataObject` to `PathBasedMetadataObject` and moved it to `authorization-common` modul.